common/playlist.h

```diff
@@ -97,6 +97,10 @@ struct playlist_entry *playlist_get_last(struct playlist *pl);
 struct playlist_entry *playlist_get_next(struct playlist *pl, int direction);
 struct playlist_entry *playlist_entry_get_rel(struct playlist_entry *e,
                                               int direction);
+struct playlist_entry *playlist_get_first_in_next_playlist(struct playlist *pl,
+                                                           int direction);
+struct playlist_entry *playlist_get_first_in_same_playlist(
+    struct playlist_entry *entry, char *current_playlist_path);
```

Comment:
Accidentally indented here.

Reply:
It's not accidental, `struct playlist_entry *playlist_get_first_in_same_playlist(struct playlist_entry *entry,` is 89 characters. Do you have a better idea to indent it?

Reply:
Oh gee, I didn't even see that. I think the normal convention for something like this is more like:
```
struct playlist_entry *playlist_get_first_in_same_playlist(struct playlist_entry *entry,
                                                                                        char *current_playlist_path);
```